### PR TITLE
RAB-1149: Add manager_authentication support

### PR DESCRIPTION
**Pull request by StevenVAIDIE**
In this PR, 

We added the support of the new authenticator manager of Symfony

### CODE REVIEW

**Review by MadeWilson — APPROVED** [review]

File: Security/Authentication/Authenticator/OAuthAuthenticator.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        }
+
+        $scope = $accessToken->getScope();
+        $user = $accessToken->getUser();
+        $username = $user instanceof UserInterface ? $user->getUserIdentifier() : '';
+
+        $userBadge = new UserBadge($username);
+
+        return new SelfValidatingPassport($userBadge, [
+            // new ScopeBadge($scope)
```
Comment (MadeWilson) [inline Security/Authentication/Authenticator/OAuthAuthenticator.php]: dead code ?

Comment (ghoussard) [reply Security/Authentication/Authenticator/OAuthAuthenticator.php]: Yes we forgot to clean it

**Review by StevenVAIDIE — COMMENTED** [review]
Remaining deprecation :

```
2022-11-30T14:36:06+00:00 [info] User Deprecated: Method "Symfony\Component\DependencyInjection\Extension\Extension::getAlias()" might add "string" as a native return type declaration in the future. Do the same in child class "FOS\OAuthServerBundle\DependencyInjection\FOSOAuthServerExtension" now to avoid errors or add an explicit @return annotation to suppress this message.
```

**Comment by ghoussard** [comment]
> Remaining deprecation :
> 
> ```
> 2022-11-30T14:36:06+00:00 [info] User Deprecated: Method "Symfony\Component\DependencyInjection\Extension\Extension::getAlias()" might add "string" as a native return type declaration in the future. Do the same in child class "FOS\OAuthServerBundle\DependencyInjection\FOSOAuthServerExtension" now to avoid errors or add an explicit @return annotation to suppress this message.
> ```

Done, thanks!

**Review by tseho — APPROVED** [review]
Tested that it does not affect Connect App.

**Pull request merged**